app/Resources/views/field_of_study/show_all.html.twig

```diff
@@ -1,53 +1,74 @@
 {% extends 'adminBase.html.twig' %}
 
 {% block breadcrumb %}
-    <li class="breadcrumb-item active">
-        Linjer
-    </li>
+	<li class="breadcrumb-item">
+		{% if is_granted_team_leader() %}
+			<a href="{{ path('show_field_of_studies') }}">
+			{% else %}
+				<a href="{{ path('show_field_of_studies') }}">
+				{% endif %}
```

Comment:
I think the indent in the if-statements is a bit off. `if`, `else`, and `endif` should have same indent.